custom_components/ha_washdata/ws_api.py

```
@@ -5405,6 +5458,187 @@ async def ws_get_dtw_debug(
+    try:
+        match_config = playground._matching_config(store)  # noqa: SLF001
+    except Exception:  # pylint: disable=broad-exception-caught
+        match_config = {}
+
+    # Classic suggestions from the store (periodic analysis results), filtered to
+    # keys the Playground actually exposes — cheap dict read, no executor needed.
+    raw_sugg: dict[str, Any] = {}
+    try:
+        raw_sugg = store.get_suggestions() or {}
+    except Exception:  # pylint: disable=broad-exception-caught
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)